### PR TITLE
avoiding math range errors in fbbt

### PR DESCRIPTION
**Pull request by michaelbynum**
## Summary/Motivation:
This PR updates the interval arithmetic for exp to avoid math range errors when performing FBBT. 

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the BSD license.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: pyomo/contrib/fbbt/interval.py [hunk 1]
```diff
@@ -282,7 +282,15 @@ def _inverse_power2(zl, zu, xl, xu, feasiblity_tol):
 
 
 def exp(xl, xu):
-    return math.exp(xl), math.exp(xu)
+    if xl >= 100:
+        lb = math.inf
+    else:
+        lb = math.exp(xl)
+    if xu >= 100:
```
Comment (jsiirola) [inline pyomo/contrib/fbbt/interval.py]: Where did 100 come from?  I would prefer to see the constant pulled from Python or with a try-except block.  Is there a similar issue for the lower limits and underflow?

Comment (michaelbynum) [reply pyomo/contrib/fbbt/interval.py]: @jsiirola Good suggestion. I went with the try-except approach. There is not a similar issue for the lower limits and underflow. If the bounds are too small (large negative numbers), then the ```exp``` function from ```math``` will just return 0.

**Comment by michaelbynum** [comment]
@jsiirola Good suggestion. I went with the try-except approach. There is not a similar issue for the lower limits and underflow. If the bounds are too small (large negative numbers), then the ```exp``` function from ```math``` will just return 0.

**Comment by blnicho** [comment]
I don't think the Jenkins test failures are related to the changes in this PR but rather because this branch is a few commits behind Pyomo master. Going to go ahead and merge.

**Pull request merged**